src/main/java/domain/Price.java

```diff
@@ -0,0 +1,53 @@
+package domain;
+
+import java.util.Objects;
+
+public class Price {
+    private static final Calculator calculator = Calculator.getInstance();
```

Comment:
계산기를 재 활용해주셨군요 😄 👍 
정적 상수로 선언하셨으니 대문자로 변경되면 더 좋을 것 같습니다 😄 
```suggestion
    private static final Calculator CALCULATOR = Calculator.getInstance();
```